doc/apiary.apib

```diff
@@ -0,0 +1,754 @@
+FORMAT: 1A
+HOST: http://localhost:8000
+
+# Device Management API
+
+Device manager is responsible for management operations (CRUD) over devices and templates. 
+
+All devices are created based on a *template*, which can be thought as a model of a device. As "model" we could think of part numbers or product models - one *prototype* from which devices are created. Templates in dojot have one label (any alphanumeric sequece), a list of attributes which will hold all the device emitted information, and optionally a few special attributes which will indicate how the device communicates, including transmission methods (protocol, ports, etc.) and message formats. 
```

Comment:
Does the template label have to be unique?

Reply:
No